lower/llpcSpirvLowerImageOp.cpp

```diff
@@ -600,6 +627,201 @@ void SpirvLowerImageOp::visitCallInst(
     }
 }
 
+// =====================================================================================================================
+// Tracks descriptor loading to handle CFG.
+llvm::Value* SpirvLowerImageOp::TrackDescriptorLoad(
```

Comment:
TrackDescriptorLoad
->
LoadImageDescriptor

Change the comments:
// Loads image descriptor from the provided load source value. Meanwhile, extract image call metadata and memory qualifier.